packages/dtslint/tslint.json

```diff
@@ -1,17 +0,0 @@
-{
-    "extends": "tslint:latest",
```

Comment:
this is, somehow, the old tslint rules for the dtslint codebase itself. This PR should also delete dt.json and dtslint.json and also isn't there dtlist-expect-only.json?

Reply:
> dtlist-expect-only.json

Err, I don't see any files or text like `/.*expect-only.*/`?

Reply:
> dtlist-expect-only.json

I removed this file when I ported it to eslint.